draft-pda-protocol.md

```diff
@@ -509,17 +508,16 @@ The leader's response to malformed requests is specified in
 Let `[helper]` denote the helper URL encoded by
 `PAUploadStartResp.param.helper_url`.  When a client sends a GET request to
 `[helper]/key_config`, the helper responds with status 200 and an `HpkeConfig`
-message. The client aborts and alerts the leader with "unsupported helper" if
-any of the following happen:
-
-* the client and helper failed to establish a secure, helper-authenticated
-  channel;
-* the GET request to the helper URL failed or didn't return a valid key config;
-  or
-* the key config specifies a KEM, KDF, or AEAD algorithm the client doesn't
+message. Similarly, when a client sends a GET request to `[leader]/key_config`,
+the leader responds with status 200 and an `HpkeConfig`. The client aborts and
+alerts the leader with "unsupported helper" if any of the following happen:
```

Comment:
Hmm.... So how does this work if the client can't connect to the leader.

Reply:
What do you mean by "feedback channel"?

Reply:
I assume he means the alert from client to leader. What is the leader actually supposed to do with that information?

Reply:
Ah yes, good call. I removed the alert.